plugin.json

```diff
@@ -88,6 +88,6 @@
         "default": false
       }
     ],
-    "footer": "Use this webhook URL format to [configure the Jira integration.](https://github.com/mattermost/mattermost-plugin-jira/blob/master/readme.md)  `https://SITEURL/plugins/jira/api/v2/webhook?secret=WEBHOOKSECRET`"
+    "footer": "Use `/jira webhook` to see fully expanded URL to [configure the Jira integration.](https://github.com/mattermost/mattermost-plugin-jira/blob/master/readme.md)"
```

Comment:
2/5  - I recommend updating this section to inform the user to run the /jira webhook command inside of a channel.  The example URL is great to show the webhook URL and the importance of  the Webhook-secret.   

https://mattermost.gitbook.io/plugin-jira/setup/configuration#step-4-required-setup-webhooks-from-jira

Perhaps even a ticket should be created if a secret is not generated the webhook displays an error?

Reply:
@Francois-D Thanks for the effort on this and https://github.com/mattermost/mattermost-plugin-jira/pull/525